test/test_utils.py

```diff
@@ -826,6 +826,10 @@ def test_external_module_register(self):
         with self.assertRaisesRegex(RuntimeError, "The runtime module of"):
             torch._register_device_module('xpu', DummyXPUModule)
 
+        # Clean up
+        delattr(torch, "xpu")
```

Comment:
Btw, this is better to be done in a teardown method though as there could be error in between failing this before the clean up could be run